iodata/basis.py

```diff
@@ -150,17 +150,29 @@ class MolecularBasis(NamedTuple):
         .. code-block:: python
 
             {
+                ### Conventions for Cartesian functions
+                # E.g., alphabetically ordered Cartesian functions.
                 (0, 'c'): ['1'],
                 (1, 'c'): ['x', 'y', 'z'],
-                # alphabetically ordered Cartesian functions
                 (2, 'c'): ['xx', 'xy', 'xz', 'yy', 'yz', 'zz'],
-                # or Wikipedia-ordered real solid spherical harmonics
-                # c = cosine-like
-                # s = sine-like
-                (2, 'p'): ['dc2', 'dc1', 'dc0', '-ds1', '-ds2'],
-                ...
+                ### Conventions for pure functions.
+                # The notation is referring to real solid spherical harmonics.
+                # See https://en.wikipedia.org/wiki/Solid_harmonics#Real_form
+                # c = cosine of the azimuthal angle
+                # s = sine of the azimuthal angle
```

Comment:
Would it be alright to add `number = magnetic quantum number` (or the correct term) here? It would be helpful to have a definition of each component of the notation.

Reply:
good point.